server/src/internalClusterTest/java/org/opensearch/versioning/ConcurrentSeqNoVersioningIT.java

```diff
@@ -729,9 +729,7 @@ public static void main(String[] args) throws Exception {
         if (args.length < 3) {
             System.err.println("usage: <file> <primaryTerm> <seqNo>");
         } else {
-            try (FileInputStream fis = new FileInputStream(args[0])) {
-                runLinearizabilityChecker(fis, Long.parseLong(args[1]), Long.parseLong(args[2]));
-            }
+            runLinearizabilityChecker(new FileInputStream(args[0]), Long.parseLong(args[1]), Long.parseLong(args[2]));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource created by the following constructor: *FileInputStream*. Currently, there are execution paths that do not contain closure statements, for example, when Long.parseLong throws an exception. Ensure the object returned by *FileInputStream()* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).